src/app/components/challenge/challengesubmit/challengesubmit.component.ts

```diff
@@ -413,21 +410,31 @@ export class ChallengesubmitComponent implements OnInit {
 
     const FORM_DATA: FormData = new FormData();
     FORM_DATA.append('status', 'submitting');
-    FORM_DATA.append('input_file', self.globalService.formItemForLabel(self.components, 'input_file').fileSelected);
+    if (!self.isSubmissionUsingUrl) {
```

Comment:
As there's gonna be one case, user will submit using URL or file upload so how about keeping this and [this](https://github.com/Cloud-CV/EvalAI-ngx/pull/242/files#diff-7ebebddad833dbfe4a95492011383387R421) together using `if and else` statement for easy understanding.